src/commands/registerCommands.ts

```diff
@@ -71,6 +72,21 @@ import { inspectVolume } from "./volumes/inspectVolume";
 import { pruneVolumes } from "./volumes/pruneVolumes";
 import { removeVolume } from "./volumes/removeVolume";
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function registerCommand(commandId: string, callback: (context: IActionContext, ...args: any[]) => any, debounce?: number): void {
+    registerCommandAzUI(
+        commandId,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        async (context, ...args: any[]) => {
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
+            ext.activityMeasurementService.recordActivity('overallnoedit');
```

Comment:
I'd rather leave it as a parameter; if we add other types of activity this is how it will be done, no need to start with an interface we know we'd have to change.